src/controllers/memberController.ts

```diff
@@ -84,6 +84,23 @@ class MembersController {
     ]);
     return isEmpty(memberInfo) ? null : memberInfo;
   }
+
+  // TODO : ADD TYPES TO RETURN FROM QUERY
+  async getMemberTags(id: string) {
+    const [memberTags] = await pool.query(
+      `SELECT * FROM MemberGroup WHERE person_id = ?`,
+      [id]
+    );
+
+    return memberTags;
+  }
+
+  async updateMemberPreferences(id: string) {
+    await pool.query(
```

Comment:
By any chance do we want to just return the entire updated member after this entry? Like Select * from Member where uuid = id(or just reuse the existing function)

Reply:
This is valid, we should do this.